internal/services/configstore/configstore.go

```diff
@@ -229,6 +236,8 @@ func (s *Configstore) setupDefaultRouter() http.Handler {
 	apirouter.Handle("/users", createUserHandler).Methods("POST")
 	apirouter.Handle("/users/{userref}", updateUserHandler).Methods("PUT")
 	apirouter.Handle("/users/{userref}", deleteUserHandler).Methods("DELETE")
+	apirouter.Handle("/users/{userref}/invitations", userOrgInvitationsHandler).Methods("GET")
+	apirouter.Handle("/users/{userref}/org_invitations/{orgref}/actions", userOrgInvitationActionHandler).Methods("PUT")
```

Comment:
I'll use `/orgs/{orgref}/invitations/{userref}/actions` since it's different than the gateway where we have an api related to the currently logged in user.